electrum/commands.py

```diff
@@ -761,12 +761,25 @@ async def verifymessage(self, address, signature, message):
         message = util.to_bytes(message)
         return bitcoin.verify_usermessage_with_address(address, sig, message)
 
+    def _get_fee_policy(self, fee, feerate):
+        if fee and feerate:
```

Comment:
```suggestion
        if fee is not None and feerate is not None:
```
setting `fee=0` is a perfectly valid setting that is not None